packages/ember-runtime/lib/mixins/array.js

```diff
@@ -638,7 +638,7 @@ const ArrayMixin = Mixin.create(Enumerable, {
 function EachProxy(content) {
   this._content = content;
   this._keys = undefined;
-  this.__ember_meta__ = null;
+  this.__ember_meta__ = undefined;
```

Comment:
I wanted to avoid making any significant changes for bugfix we intend to backport without proper measurement.

But I can gladly make the change